### PR TITLE
Add Memberships renewal sidebar

### PR DESCRIPTION
**Pull request by thibaudgg**
This new sidebar provides a quick way to renew all current fiscal year 
memberships that have been flagged for renewal.

- [x] Manage new basket_size/depot price changes

screenshot 2018-10-28 at 11 15 36

### CODE REVIEW

File: config/locales/active_admin.yml [hunk 1]
```diff
@@ -379,6 +385,40 @@ _:
...
+      index:
+        all_renewed:
+          one:
+            _de: "%{count_link} Abonnement %{year} wurde bereits verlängert."
+            _fr: "%{count_link} abonnement %{year} a déjà été renouvelé."
+          other:
+            _de: "%{count_link} Abonnements %{year} wurden bereits verlängert."
+            _fr: "%{count_link} abonnements %{year} ont déjà été renouvelés."
+        no_renewals:
+          _de: Kein Abonnement zur Verlängerung.
```
Comment (mrast) [inline config/locales/active_admin.yml]: Kein Abonnement zu verlängern.

File: config/locales/active_admin.yml [hunk 2]
```diff
@@ -379,6 +385,40 @@ _:
...
+            _de: "%{count} Abonnement verlängern"
+            _fr: Renouveler %{count} abonnement
+          other:
+            _de: "%{count} Abonnements verlängern"
+            _fr: Renouveler %{count} abonnements
+        renew_confirm:
+          _de: Sind Sie mehr als sicher?
+          _fr: Êtes-vous sûr de sûr?
+        renewing:
+          _de: Verlängerungen laufen...
```
Comment (mrast) [inline config/locales/active_admin.yml]: Verlängerungen am laufen...

File: config/locales/active_admin.yml (hunk 1 above)
Comment (thibaudgg) [reply config/locales/active_admin.yml]: 👍

File: config/locales/active_admin.yml (hunk 2 above)
Comment (thibaudgg) [reply config/locales/active_admin.yml]: :+1:

**Pull request merged**